src/40_Dynamic_AMP/Multi_Page_Flow.html

```diff
@@ -0,0 +1,537 @@
+          <i class="step-dot"></i>
+        </div>
+        <button on="tap:AMP.pushState({ currentPage: currentPage + 1 })"
+                class="button-next"
+                >next</button>
+      </div>
+    </section>
+    <section hidden [hidden]="currentPage != 1"> 
+      <div class="top-bar">
+        Step 2
+      </div>
+      <div class="content">Here is some more content ...</div>
+      <div class="bottom-bar">
+        <button class="button-prev" on="tap:AMP.pushState({ currentPage: currentPage - 1 })">back</button>
+        <div class="step-dots">
+          <i class="step-dot active"></i>
+          <i class="step-dot active"></i>
+          <i class="step-dot"></i>
+        </div>
+        <button on="tap:AMP.pushState({ currentPage: currentPage + 1 })"
+                class="button-next"
+                >next</button>
+      </div>
+    </section>
+    <section hidden [hidden]="currentPage != 2"> 
+      <div class="top-bar">
+        Step 3
+      </div>
+      <div class="content">Done!</div>
+      <div class="bottom-bar">
+        <button class="button-prev" on="tap:AMP.pushState({ currentPage: currentPage - 1 })">back</button>
+        <div class="step-dots">
+          <i class="step-dot active"></i>
+          <i class="step-dot active"></i>
+          <i class="step-dot active"></i>
+        </div>
+      </div>
+    </section>
+  </div>
+  <h2 class="m0 p1 pt2">A vertical Stepper</h2>
+  <div class="p1">Use the browser back button to go to the previous view.</div>
+<!-- ## A vertical Stepper -->
+<!-- Vertical steppers work well if steps depend on each other. The stepper is implemented similar to the first sample using a 
+  variable `currentStep` to keep track of the currently active step. We additionally define a step title button which is always visible 
+  and shows the status of the current step. The title needs to reflect three different states (active, complete, disabled). To avoid too complex amp-bind expressions the three states are split into three different bindings:    
+
+  * The title class gets updated if the current step is active.
+  * The title gets a `disabled` attribute if the **previous** step has not yet been completed
+  * The nested icon's class gets set to `step-complete` or `step-incomplete` based on whether the step has finished. 
+
+Clicking on the title will go to the corresponding step (if already possible):
+
+  ```html
+  <button class="step-title" 
+      [class]="currentStep != 1 ? 'step-title' : 'step-title step-active'" 
+       disabled [disabled]="!animalSelected"
+       on="tap:AMP.pushState({ currentStep: 1 })">
+    <i class="step-incomplete" 
+       [class]="colorSelected ? 'step-complete' : 'step-incomplete'" data-step-nr="2"></i>
+       Color
+  </button>
+  ```
+
+Another thing that's new in this sample is the continue buttons is disabled by default. Only when the step is completed (in this case when a selection has been made), the button will be enabled:
```

Comment:
Another thing that's new in this sample is the continue buttons is disabled by default... -> The "Continue" button is disabled by default and only...

This is a personal preference! Because when you say "Another thing that's new in this sample..." I don't understand what you are comparing it to. Also, our samples have no concept of new and old, they don't have dates! Imagine a person going to ABE for the first time and opening this sample, what "new" means? I think that would make more sense into a blog post where you have the concept of new. But again, that's personal opinion.

Reply:
good point! done